content/en/news/security/istio-security-2021-003/index.md

```diff
@@ -0,0 +1,134 @@
+---
+title: ISTIO-SECURITY-2021-003
+subtitle: Security Bulletin
+description: Potential misuse of mTLS-only fields in AuthorizationPolicy with plain text traffic. 
+cves: [N/A]
+cvss: "N/A"
+vector: ""
+releases: ["All releases 1.5 and later"]
+publishdate: 2021-04-15
+keywords: [CVE]
+skip_seealso: true
+---
+
+{{< security_bulletin >}}
+
+This is a security advisory for customers to check the authorization policy to make sure [mTLS (STRICT mode) is enabled](/docs/tasks/security/authentication/authn-policy/#globally-enabling-istio-mutual-tls-in-strict-mode)
+when using [mTLS-only fields](/docs/concepts/security/#dependency-on-mutual-tls) in the authorization policy.
+
+You can stop reading if
+
+- Your authorization policy does not use [mTLS-only fields](/docs/concepts/security/#dependency-on-mutual-tls); or
+
+- Your authorization policy uses mTLS-only fields and you have also enabled mTLS with STRICT mode or your authorization
+policy is configured to reject plain text traffic explicitly.
+
+## Issue
+
+In authorization policy, the following are mTLS-only fields:
+
+- the principals field under the source section
+- the namespaces field under the source section
```

Comment:
The examples also show `notNamespaces`. Should this be called out in addition to the `namespaces` field?

Reply:
good suggestion, I actually updated the example to make it shorter and also extended the explanation to make it more clear.